client/src/hooks/useMetadataOnly.js

```diff
@@ -0,0 +1,19 @@
+import { filterWhere } from 'helpers/filterWhere'
+
+// TODO: Implement the logic to handle multiple metadata files (i.e. project and portal-wide) later
+export const useMetadataOnly = (computedFiles) => {
+  const metadataComputedFile = filterWhere(computedFiles, {
```

Comment:
```suggestion
  const metadataComputedFiles = filterWhere(computedFiles, {
```

Since there are multiple we should have the variable name reflect that.

Reply:
See the commit at 06be077ebea7c8e7c84ac5e5d90f71a377cac85a.